modules/distr-tracing-otel-install-web-console.adoc

```diff
@@ -0,0 +1,119 @@
+////
+This module included in the following assemblies:
+- distr_tracing_otel/distr-tracing-otel-installing.adoc
+////
+
+:_content-type: PROCEDURE
+[id="distr-tracing-install-otel-operator_{context}"]
+= Installing the {OTELShortName} by using the OpenShift web console
+
+You can install the {OTELShortName} by using the *Administrator* view of the OpenShift web console.
+
+.Prerequisites
+
+* You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role. For {product-dedicated}, you must be logged in using an account with the `dedicated-admin` role.
+
+* An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
++
+[TIP]
+====
+* Ensure that your OpenShift CLI (`oc`) version is up to date and matches your {product-title} version.
+
+* Run `oc login`:
++
+[source,terminal]
+----
+$ oc login --username=<your_username> 
+----
+====
+
+.Procedure
+
+. Install the {OTELOperator}:
+
+.. Go to *Operators* -> *OperatorHub* and search for `{OTELOperator}`.
+
+.. Select the *{OTELOperator}* that is *provided by Red Hat* -> *Install* -> *Install* -> *View Operator*.
++
+[IMPORTANT]
+====
+This installs the Operator with the default presets:
+
+* *Update channel* -> *stable*
+* *Installation mode* -> *All namespaces on the cluster*
+* *Installed Namespace* -> *openshift-operators*
+* *Update approval* -> *Automatic*
+====
+
+.. In the *Details* tab of the installed Operator page, under *ClusterServiceVersion details*, verify that the installation *Status* is *Succeeded*.
+
+. Create an OpenShift project of your choice for the *OpenTelemetry Collector* instance that you will create in the next step: go to *Home* -> *Projects* -> *Create Project*.
```

Comment:
We use the `. <text>: Some step text` convention to indicate optional or conditional steps, so would recommend against using it here.

```suggestion
. Create an OpenShift project of your choice for the *OpenTelemetry Collector* instance that you will create in the next step by going to *Home* -> *Projects* -> *Create Project*.
```

Reply:
Thank you. I'm learning something here. Applied your suggestion manually.